### PR TITLE
Added a forge link just in case

### PR DESCRIPTION
**Pull request by renpenguin**
Added a link directly to the forge 1.12.2 download page on the README

### CODE REVIEW

**Comment by ghost** [comment]
kek

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Find our plugins [here](https://github.com/lambda-plugins).
 
 ## Installation
 1. Install Minecraft 1.12.2
-2. Install Forge
+2. Install Forge latest installer [here](https://files.minecraftforge.net/net/minecraftforge/forge/index_1.12.2.html)
```
Comment (mmvanheusden) [inline README.md]: `Install the latest Forge for 1.12.2 [(download)](https://files.minecraftforge.net/net/minecraftforge/forge/index_1.12.2.html)` Sounds better to me

**Review by mmvanheusden — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: was gonna say ^

Comment (Avanatiker) [reply README.md]: +1

**Pull request merged**